PiicoDev_Unified.py

```diff
@@ -37,6 +45,12 @@ def read16(self, addr, nbytes, stop=True):
 
     def __init__(self, bus=None, freq=None, sda=None, scl=None):
         raise NotImplementedError("__init__")
+    
+    def scan(self):
```

Comment:
I'd suggest adding some docstrings so that ppl can determine what these functions do while in the editor (or in this case that they're unsupported here) I know it seems obvious stating this right above a NotImplementedError exception, but this way it'll work between files in the appropriate IDE:

```python
def scan(self):
    '''
    Function not implemented in PiicoDev_Unified, see _Link to README.md_
    ''''
    raise NotImplementedError("scan")
```